src/top-issues/schemas/topIssues.schema.ts

```diff
@@ -0,0 +1,22 @@
+import { createZodDto } from 'nestjs-zod';
+import { z } from 'zod';
+
+export const UpdateTopIssueSchema = z.object({
+  name: z.string(),
```

Comment:
Where's the `id` property here? Updates should be done via `id`, no?

Reply:
That's in the path v1/topIssues/:id